ext/drsyms/CMakeLists.txt

```diff
@@ -84,14 +84,14 @@ if (WIN32)
   # We don't need a stamp file b/c the .lib is not a source; plus a stamp
   # file causes Ninja to fail to build.
   add_custom_command(OUTPUT "${dbghelp_lib}"
-    DEPENDS "${dbghelp_src}" "${dbghelp_def}"
-    COMMAND "${CMAKE_C_COMPILER}" ARGS /nologo /c /Ob0 ${dbghelp_src}
-    COMMAND "${LIB_EXECUTABLE}" ARGS
-      /nologo /name:dbghelp.dll /def:${dbghelp_def}
-      ${CMAKE_CURRENT_BINARY_DIR}/dbghelp_imports.obj
-    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
-    VERBATIM # recommended: p260 of cmake book
-    )
+          DEPENDS "${dbghelp_src}" "${dbghelp_def}"
```

Comment:
Please undo all of these indentation changes.  Our CMake indentation is 2 spaces and it was already what we wanted.

Reply:
@derekbruening Fixed.